src/dex-ui/pages/dao/CreateADAO/forms/DAODetailsForm.tsx

```diff
@@ -27,36 +57,43 @@ export function DAODetailsForm() {
         isInvalid={Boolean(errors.name)}
         errorMessage={errors.name && errors.name.message}
       />
-      {/* TODO: For a future version of the form.
-        <FormInput<"description">
+      <FormTextArea<"description">
+        textAreaProps={{
+          id: "description",
+          label: "Description",
+          placeholder: "Add a description for your DAO",
+          register: {
+            ...register("description", {
+              required: { value: true, message: "A description is required." },
+              validate: (value) => value.length > 239 || "At-least 240 characters is required.",
```

Comment:
1) `At-least` => `At least`
2) The 240 character count is a `max` character limit not a `min`.

Reply:
My bad, its done.